active_directory_ldap/help.md

```diff
@@ -486,6 +486,7 @@ the query results, and then using the variable step $item.dn
 
 # Version History
 
+* 5.0.0 - Add new input to connection to allow for Auto Referrals | A number of bug fixes
```

Comment:
let's get the internal language out of this - most users aren't going to understand what "auto referrals" means - how about something like "Expose option to enable referral chasing to support multi-domain environments" instead?

Reply:
Also, should we call out that this is a fundamental change in the scope of the plugin?  It means the plugin WILL CONNECT TO DOMAIN CONTROLLERS NOT SPECIFIED IN THE CONFIG